src/views/NewFlow.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="new-contents">
+    <div v-if="activeFlow" class="row">
+      <div class="col-sm-8 offset-sm-2">
+        <div class="card">
+          <div class="card-body">
+            <div v-if="flowError" class="alert alert-danger" role="alert">
+              {{flowError | trans}}
+            </div>
+            <flow-editor :flow="activeFlow" flow-header="flow-builder.create-flow" :sidebar=false />
+
+            <div class="float-right">
+              <router-link :to="route('trees.editTree', {treeId: activeFlow.uuid, component: 'interaction-designer', mode: 'edit'})"
+                class="btn btn-primary"
+                event=""
+                @click.native.prevent="handlePersistFlow(route('trees.editTree', {treeId: activeFlow.uuid, component: 'interaction-designer', mode: 'edit'}))">
```

Comment:
Does this need to be a router-link? It seems like we prevent the router-link from doing its natural thing and do our own routing below. Should this just be a vanilla button?